docs/reference/chart-font.md

```diff
@@ -1,3 +1,6 @@
+---
+meta_description: "Image-charts API propose to customize your graphics as for example with chart font. This documentation shows you how to use the API url parameters to generate a chart."
```

Comment:
- "propose to" pas anglais
- "Image-Charts supports more than 900+ fonts! Learn how to customize your chart fonts."